vision/detect/detect.go

```diff
@@ -14,22 +14,24 @@ import (
 	"io"
 	"os"
 
-	"cloud.google.com/go/vision"
+	vs "cloud.google.com/go/vision/apiv1"
```

Comment:
cloud.google.com/go/vision/apiv1 package name is already vision. If we keep the snippets with "vision", users are likely to import the wrong package when they copy paste from the docs: http://imgur.com/a/jizUo.  Use of "vs" disables goimports and make them explicitly import the package by looking at snippet's import path.